src/blog/2023/01/3-quick-node-red-tips-1.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 3 Quick Tips for using Node-RED
+subtitle: Save yourself time when working on Node-RED with these three tips.
+description: Save yourself time when working on Node-RED with these three tips.
+date: 2023-02-07 18:00:00
+authors: ["rob-marcer"]
+---
+
+There is usually more than one way to complete a given task in software and Node-RED is no exception. In this blog post we are going to share three useful tips to save yourself time when working on your flows.
+<!--more-->
+
+### 1. Use control+left-click to search your nodes
+
+Sometimes it's quicker to search for a node using its name rather than scrolling through the palette. Simply hold control then left-click to bring up a searchable list.
+
+![Select a node without having to use the palette](../images/load-node.gif)
+
+### 2. Split sections of your code using the link nodes
+
+If you want to separate your flow into two distinct sections, link nodes are a great way to format your work. As we covered in our blog on [Node-RED best practices](https://flowforge.com/blog/2022/12/node-red-flow-best-practice), the combination of link nodes and grouped flows is very powerful.
+
+To split your flow select the input and output nodes then right click, select 'Show Action List' and then type 'split'. Select 'Split wire with link nodes'.
+
+![Insert a new node between two existing node](../images/split-with-link.gif)
+
+### 3. Link multiple inputs and outputs in one command
+
+Once a switch node has several outputs it can be slow to manually wire each to the new node. Using the action menu (right click), select 'Show Action List' then 'Wire Node to Multiple' this option will join everything up in one step.
+
+![Link multiple inputs and outputs in one command](../images/join-wires.gif)
```

Comment:
I've learned yesterday that this image won't have an image alt tag, which can be added like:

```suggestion
![Link multiple inputs and outputs in one command](../images/join-wires.gif "Quickly link multiple nodes in Node-RED")
```

Reply:
I've added in the alt tags for all the images, no idea why the content of the [] does not populate the alt, it should really.